tests/Unit/Evolution/DiscontinuousGalerkin/Actions/Test_NormalCovectorAndMagnitude.cpp

```diff
@@ -189,11 +189,25 @@ void check_normal_covector_quantities(
       }
     }
 
-    evolution::dg::Actions::detail::
-        unit_normal_vector_and_covector_and_magnitude<tmpl::conditional_t<
-            UseFlatSpace, FlatSpaceSystem, SystemWithInverseMetric<Dim>>>(
-            box, make_not_null(&fields_on_face), direction,
-            unnormalized_normal_covectors);
+    db::mutate<
+        evolution::dg::Tags::InternalFace::NormalCovectorAndMagnitude<Dim>>(
+        box,
```

Comment:
Any chance this could be super classy and just use the detail function as the callable argument instead of the lambda?

Reply:
Sure, I've done this and rebased on develop. I added a fixup so that you and the other reviewers can easily see the change (mostly trivial but not completely)